lib/ssf/ssf_methods.rb

```diff
@@ -58,6 +58,16 @@ def set_name(name)
     end
 
 
+    def set_tag(name, value)
+      self.tags[name] = value
+    end
+
+    def set_tags(tags)
+      if tags.is_a?(Hash)
```

Comment:
The cynic in me wants to add a test for that, but I don't know if that's overkill